contracts/test/userReplicaSetManager.js

```diff
@@ -98,38 +94,48 @@ contract('UserReplicaSetManager', async (accounts) => {
 
         userReplicaSetManager = await UserReplicaSetManager.at(proxyContractDeployTx.address)
 
+        let seedComplete = await userReplicaSetManager.getSeedComplete({ from: userReplicaBootstrapAddress })
+        assert.isFalse(seedComplete, "Expect no seed operation")
+        let seedTx = await userReplicaSetManager.seedBootstrapNodes(
+            bootstrapSPIds,
+            bootstrapDelegateWallets,
+            { from: userReplicaBootstrapAddress }
+        )
+        seedComplete = await userReplicaSetManager.getSeedComplete({ from: userReplicaBootstrapAddress })
+        assert.isTrue(seedComplete, "Expect completed seed operation")
+
         // Confirm constructor events were fired as expected
         await expectEvent.inTransaction(
-            proxyContractDeployTx.transactionHash,
+            seedTx.tx,
             UserReplicaSetManager,
             'AddOrUpdateContentNode',
             {
-                _newCnodeId: toBN(cnode1SpID),
-                _newCnodeDelegateOwnerWallet: cnode1Account,
+                _cnodeSpId: toBN(cnode1SpID),
+                _cnodeDelegateOwnerWallet: cnode1Account,
                 _proposer1Address: addressZero,
                 _proposer2Address: addressZero,
                 _proposer3Address: addressZero
            }
         )
         await expectEvent.inTransaction(
-            proxyContractDeployTx.transactionHash,
+            seedTx.tx,
             UserReplicaSetManager,
             'AddOrUpdateContentNode',
             {
-                _newCnodeId: toBN(cnode2SpID),
-                _newCnodeDelegateOwnerWallet: cnode2Account,
+                _cnodeSpId: toBN(cnode2SpID),
+                _cnodeDelegateOwnerWallet: cnode2Account,
                 _proposer1Address: addressZero,
                 _proposer2Address: addressZero,
                 _proposer3Address: addressZero
            }
```

Comment:
why not also assert the `_proposerSpIds` field in these checks?

Reply:
https://github.com/AudiusProject/audius-protocol/pull/1058/files#diff-a6a6beb4e223be0e7e08b13032cc4eac0c5974822f5e5f37d408c9d2d61e4752R29 covered here, OZ did not like the array types.

Reply:
yes, however it is covered in this test file in several places if you read through the test cases. adding here too, just want to note that this is 100% covered in the non-bootstrap events

Reply:
oh lol, you right. it def is